docs/glossary.md

```diff
@@ -0,0 +1,60 @@
+---
+id: glossary
+title: GLOSSARY
+sidebar_label: Glossary
+slug: glossary
+---
+
+# 🤸Shenanigan's Glossary🤸
+
+1. **APY:** The annual percentage yield (APY) is the real rate of return earned on a savings deposit or investment taking into account the effect of compounding interest.
```

Comment:
I personally think anything that has to do with the general ethereum scope shouldn't be here. That way we cut down on bloat

Reply:
I thought we were bringing normal people into the space? I think having a seperate glossary with web 3 terms is good as kind of like an educational platform as well. I spoke with @malikvprice  and we both agreed this was a good idea